anitya/tests/test_flask_api_v2.py

```diff
@@ -35,6 +35,475 @@ def _read_json(output):
     return json.loads(output.get_data(as_text=True))
 
 
+class PackagesResourceGetTests(DatabaseTestCase):
+    """Tests for HTTP GET on the ``api/v2/packages/`` resource."""
+
+    def setUp(self):
+        super(PackagesResourceGetTests, self).setUp()
+        self.app = self.flask_app.test_client()
+        session = Session()
+        self.user = models.User(email='user@example.com', username='user')
+        self.api_token = models.ApiToken(user=self.user)
+        fedora = models.Distro('Fedora')
+        debian = models.Distro('Debian')
+        jcline_linux = models.Distro('jcline linux')
+        session.add_all([self.user, self.api_token, fedora, debian, jcline_linux])
```

Comment:
where can i download jcline_linux? how does it compare to [dros](https://github.com/jeremycline/dros)?